File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/favorite_detail_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class FavoriteDetailVO:
    """收藏详情响应"""
    created_at: str = None
    updated_at: str = None
    favorite_id: str = None
    favorite_type: str = None
    target_id: str = None
    favorite_name: str = None
    favorite_description: str = None
    folder_id: str = None
    folder_name: str = None
    tags: str = None
    favorite_time: str = None
    update_time: str = None
    target_info: Any = None

File: sdkwork-app-sdk-typescript/src/types/favorite-detail-vo.ts
/** 收藏详情响应 */
export interface FavoriteDetailVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 收藏ID */
  favoriteId?: string;
  /** 收藏类型 */
  favoriteType?: string;
  /** 收藏对象ID */
  targetId?: string;
  /** 收藏名称 */
  favoriteName?: string;
  /** 收藏描述 */
  favoriteDescription?: string;
  /** 文件夹ID */
  folderId?: string;
  /** 文件夹名称 */
  folderName?: string;
  /** 标签 */
  tags?: string;
  /** 收藏时间 */
  favoriteTime?: string;
  /** 更新时间 */
  updateTime?: string;
  /** 收藏对象信息 */
  targetInfo?: unknown;
}
